apps/web/utils/actions/group.ts

```
@@ -13,216 +13,227 @@ import { findNewsletters } from "@/utils/ai/group/find-newsletters";
+    if (!session?.user.id) return { error: "Not logged in" };
+
+    const group = await prisma.group.findUnique({
+      where: { id: body.groupId },
+    });
+    if (!group) return { error: "Group not found" };
+    if (group.userId !== session.user.id)
+      return { error: "You don't have permission to add items to this group" };
+
+    await prisma.groupItem.create({ data: addGroupItemBody.parse(body) });
+
+    revalidatePath(`/automation`);
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)